### PR TITLE
Vectorised initialisation for Floyd-Warshall

### PR DESCRIPTION
**Pull request by Alessi0X**
Regarding issue #27 

PS: sorry for opening the PR on your repo earlier. My bad.
PS2: I'm rather new to GitHub, so let me know if everything's ok

### CODE REVIEW

File: grakel/graph.py [hunk 1]
```diff
@@ -1714,14 +1715,9 @@ def floyd_warshall(adjacency_matrix):
...
     # Initialization
-    for i in range(0, n):
-        for j in range(0, n):
-            if (i == j):
-                dist[i, j] = 0.
-            elif (adjacency_matrix[i, j] == 0):
-                dist[i, j] = float("Inf")
-            else:
-                dist[i, j] = adjacency_matrix[i, j]
+    dist = copy.deepcopy(adjacency_matrix)
```
Comment (gperakis) [inline grakel/graph.py]: Hello there!
I've tried to run both old and new implementation.
The results are the same if the adjacency matric is a matrix with floats. But it breaks when the input are integers

I've tried both with 
x1 = np.random.randint(0, 10, (10, 10))
x2 = np.random.randint(0, 10, (10, 10)).astype(float)

x2 passes and returns the same results as the old. 
x1 raises overflow error

Comment (ysig) [reply grakel/graph.py]: Does np.inf solve this problem instead of float("Inf").
Can you please have a look?

Comment (ysig) [reply grakel/graph.py]: It is seems it is not easily solved.
https://stackoverflow.com/questions/56811682/how-can-i-convert-np-inf-to-an-integer-type

@Alessi0X can you please provide a workaround that supports integers in a new pull-request?

**Comment by ysig** [comment]
Tried a fix in last commit:

https://github.com/ysig/GraKeL/commit/2c8d6124af5dec7a6431c0cec58a5b0696aa0329

**Pull request merged**